usaspending_api/etl/management/commands/load_submission.py

```diff
@@ -207,6 +207,13 @@ def get_or_create_program_activity(row, submission_attributes):
     if prg_activity is None and row['program_activity_code'] is not None:
         prg_activity = RefProgramActivity.objects.create(**filters)
         logger.warning('Created missing program activity record for {}'.format(str(filters)))
+    # If the PA has a blank name, update it with the value in the row.
+    # PA loader should overwrite the names for the unique PAs from the official
+    # domain values list if the title needs updating, but for now grab it from the submission
+    if prg_activity and not prg_activity.program_activity_name:
```

Comment:
I don't _think_ so because the names may vary slightly across submissions and we don't want to create duplicate program activities for slightly different names. I definitely got blank PA names when loading everything fresh from the `misc-submission` branch so I don't think it was addressed before this.